@xen-orchestra/backups/docs/VM backups/README.md

```diff
@@ -69,6 +69,8 @@ job.start(data: { mode: Mode, reportWhen: ReportWhen })
 ├─ task.warning(message: string)
 ├─ task.start(data: { type: 'VM', id: string })
 │  ├─ task.warning(message: string)
+|  ├─ task.start(message: 'clean-vm')
+│  │  └─ task.end
```

Comment:
Any way to link this task to a specific remote?

What if we move it under the `export` task in the future, will it break the UI?

Reply:
After discussion, moving the task under the `export` task will not break the UI.

Therefore, this PR is good enough to go now.